Makefile

```diff
@@ -440,6 +440,11 @@ build/toolchain/bin/golangci-lint$(EXE_EXTENSION):
 	mv $(TOOLCHAIN_DIR)/temp-golangci/golangci-lint$(EXE_EXTENSION) $(TOOLCHAIN_BIN)/golangci-lint$(EXE_EXTENSION)
 	rm -rf $(TOOLCHAIN_DIR)/temp-golangci/
 
+build/toolchain/python/:
+	# mkdir -p $(TOOLCHAIN_DIR)/python/
```

Comment:
Create the python directory.

Reply:
using virtualenv will automatically create the python directory